tsconfig.json

```diff
@@ -14,10 +14,8 @@
     "noUnusedLocals": true,
     "noUnusedParameters": true,
     "noImplicitReturns": true,
-    "noFallthroughCasesInSwitch": true
+    "noFallthroughCasesInSwitch": true,
+    "types": ["node", "mocha"]
```

Comment:
remove this and just install `@types/node`, `@types/mocha` if they're not already. we should never have a `types` set